src/js/rtc_session.js

```diff
@@ -375,6 +401,12 @@ export class TalkingState extends RTCSessionState {
         this._rtcSession._signalingChannel.hangup();
         this.transit(new DisconnectedState(this._rtcSession));
     }
+    onIceStateChange(evt) {
+        if (evt.currentTarget.iceConnectionState == 'disconnected') {
```

Comment:
is this the only state change we want to monitor, how about logging all state changes will that be too much?